wodles/gcloud/buckets/bucket.py

```diff
@@ -193,31 +215,26 @@ def process_data(self):
         int
             Number of blobs processed.
         """
-        def get_last_creation_time():
-            """Get the latest creation time value stored in the database for the given project, bucket_name and
-            prefix."""
-            creation_time = datetime.min.replace(tzinfo=pytz.UTC)
-            query_creation_time = self.db_connector.execute(
-                self.sql_find_last_creation_time.format(table_name=self.db_table_name,
-                                                        project_id=self.project_id,
-                                                        bucket_name=self.bucket_name,
-                                                        prefix=self.prefix))
-            try:
-                creation_time = query_creation_time.fetchone()[0]
-                creation_time = datetime.strptime(creation_time, self.datetime_format)
-            except (TypeError, IndexError):
-                pass
-            return creation_time
+        def block_process_blob():
```

Comment:
This function is not required if the `get_last_creation_time` function is not able to return a `None` value